.github/workflows/ci.yml

```diff
@@ -4,144 +4,88 @@ on:
   push:
     branches:
       - master
-    tags:
-      - v[0-9]+.[0-9]+.[0-9]+
+      - main
+      - "v*"
   pull_request:
 
 env:
   NODE_VERSION: 10
-  PERCY_PARALLEL_NONCE: ${{ github.run_id }}-${{ github.run_number }}
-  PERCY_PARALLEL_TOTAL: 1
+  FORCE_COLOR: 1
 
 jobs:
   lint:
-    name: Lint files
+    name: Linting
     runs-on: ubuntu-latest
-    timeout-minutes: 3
```

Comment:
Can you revert deleting `timeout-minutes`? The default value is 360 so I think setting a low value is a good practice.

Reply:
Like I said in the description I am using a template for the GitHub CI that I was given. I happen to like to use the defaults wherever possible so that it keeps the code simple and gives less cognitive load for anyone trying to figure out what they're looking at.

Would you mind if I continue without adding a `timeout-minutes` on this basis?